### ISSUE TITLE
Updating via Python

### ISSUE THREAD

**Issue by scottapotamus** [microsoftgraph/microsoft-graph-docs#6055]
How would I go about doing a PATCH using Python?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 615020bb-68a9-5758-d7ee-2aec0a91e347
* Version Independent ID: 12745110-a0e8-be7d-3ae0-dc37ceb6181d
* Content: [Update contact - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/contact-update?view=graph-rest-1.0&tabs=http#feedback)
* Content Source: [api-reference/v1.0/api/contact-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/contact-update.md)
* Product: **outlook**
* Technology: **microsoft-graph**
* GitHub Login: @angelgolfer-ms
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by lebokus** [comment]
Hi, are there any news?
I try to update a Contact and get a 405 Response saying: "The OData request is not supported."

**Comment by scottapotamus** [comment]
Hey lebokus. Say you wanted to update something like an email address and assuming you've got the OAuth2Session and token sorted you can just do the following:

`
import json

graph_client = OAuth2Session(token=token)
contact_url = 'https://graph.microsoft.com/v1.0/me/contacts'
payload =  "emailAddresses": {[
          {
            "name": EMAIL NAME),
            "address": NEW EMAIL ADDRESS)
          }
        ]},
headers = {
        'authorization': 'bearer {}'.format(token),
        'Content-Type': 'application/json'
    }
graph_client.patch(url=contact_url,data=json.dumps(payload),headers=headers)
`

Basically whenever you want to update something you use graph_client.patch()

Hope this helps!
